src/LisExpr.chpl

```diff
@@ -61,57 +65,17 @@ module LisExpr
         var lv: lvtype;
         
         /* initialize the value and the vtype */
-        proc init(val: ?vtype) {
+        // Use an `in` intent to deep copy the list class
+        // rather than just store a pointer to the old list
+        proc init(in val: ?vtype) {
```

Comment:
i wanted to just store a pointer... to avoid the deep copy while parsing. I thought ownership would get transferred, maybe this does need to be `shared` to get the behavior I wanted. Everything in Python, C, and Lisp is effectively copied by ref... you only get a deep copy when you explicitly ask for it.